### PR TITLE
doc: update nested virtualization user guide

### PR DESCRIPTION
**Pull request by ZideChen0**
Tracked-On: #5923
Signed-off-by: Zide Chen <zide.chen@intel.com>

### CODE REVIEW

File: doc/tutorials/nvmx_virtualization.rst [hunk 1]
```diff
@@ -101,39 +101,47 @@ with these settings:
...
-
    - Edit :option:`vm.cpu_affinity.pcpu_id` to assign ``pCPU`` IDs to run the Service VM. If you are
      using debug build and need the hypervisor console, don't assign
      ``pCPU0`` to the Service VM.
 
-     You may need to manually edit the ACRN scenario XML configuration file to edit the ``pcpu_id`` for the Service VM (SOS):
-
      .. code-block:: xml
-        :emphasize-lines: 5,6,7
+        :emphasize-lines: 5,6,7,10,11
```
Comment (dbkinder) [inline doc/tutorials/nvmx_virtualization.rst:142]: Let's put the note back in above this code block saying:
```
.. note:: Normally you'd use the configuration tool GUI to edit the scenario XML file.
   The tool wasn't updated in time for the v2.5 release, so you'll need to manually edit
   the ACRN scenario XML configuration file to edit the ``pcpu_id`` , ``guest_flags``,
   ``legacy_vuart``, and ``console_vuart`` settings for the Service VM (SOS), as shown below:
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
We need to emphasize that the normal model for making scenario XML file changes is using the configuration tool GUI, not by editing the XML files.  (We can update this document once the configuration tools is brought up to date.)

**Comment by dbkinder** [comment]
@wenlingz @NanlinXie CI failure for doc-only change

File: doc/tutorials/nvmx_virtualization.rst (hunk 1 above)
Comment (ZideChen0) [reply doc/tutorials/nvmx_virtualization.rst:142]: Thanks.

**Review by dbkinder — APPROVED** [review]
Thanks! LGTM

**Pull request merged**